containers/CSharp/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet/.devcontainer/base.Dockerfile
+
+# [Choice] .NET version: 5.0, 3.1, 2.1
+ARG VARIANT="5.0"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
+
+# [Choice] Node.js version: none, lts/*, 16, 14, 12, 10
+ARG NODE_VERSION="none"
```

Comment:
>"none" [](http://example.com/codeflow?start=17&length=6)

is there a default version that we should offer or do we always want the user to pick one?